File: lib/core/entities/inference_payload.dart
import '../enums/inference_strategies.dart';
import '../enums/inference_type.dart';

class InferencePayload {
  final InferenceType type;
  final String model;
  final InferenceStrategy inferenceStrategy;

  InferencePayload({
    required this.type,
    required this.model,
    this.inferenceStrategy = InferenceStrategy.none,
  });
}

File: android/src/main/kotlin/com/example/flutter_flow/entities/InferencePayload.kt
package com.example.flutter_flow.entities

import com.example.flutter_flow.enums.InferenceStrategy
import com.example.flutter_flow.enums.InferenceType

class InferencePayload(val model: String,
                       val type: InferenceType,
                       val strategy: InferenceStrategy) {
    
}